examples/aws_request.ipynb

```diff
@@ -64,10 +64,11 @@
     "    \n",
     "    https://roda.sentinel-hub.com/sentinel-s2-l1c/tiles/1/C/CV/2017/1/14/0/\n",
     "\n",
-    "\n",
+    "- Automatic search by a tile id or by location and time interval using Sentinel Hub Catalog API. More examples are available in [this notebook](./data_search.ipynb).    \n",
+    "    \n",
     "- Automatic search by location and time interval using Sentinel Hub Web Feature Service (WFS):\n",
     "\n",
-    "(Note: For this functionality Sentinel Hub instance ID has to be configured according to [Configuration paragraph](http://sentinelhub-py.readthedocs.io/en/latest/configure.html#sentinel-hub-capabilities).)"
+    "  (Note: For this functionality Sentinel Hub instance ID has to be configured according to [Configuration paragraph](http://sentinelhub-py.readthedocs.io/en/latest/configure.html#sentinel-hub-capabilities).)"
```

Comment:
This is actually intentional so that it is equally indented as the bullet point 1 line above. This way markdown will show it in the same "bullet point chapter".